### PR TITLE
Added retry count for OCR validation

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-709

### Change description ###
This config parameter will be used when retrying to send OCR validation request in case of failure (5xx, etc)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -218,3 +218,7 @@ process-payments:
   enabled: ${PROCESS_PAYMENTS_ENABLED:true}
 
 use-wrapping-zip: true
+
+service-config:
```
Comment (aliveni) [inline src/main/resources/application.yaml:222]: is it a service-specific config?

Comment (apereverzin) [reply src/main/resources/application.yaml:222]: no, it is not. is there a better config for this situation?

Comment (rkondratowicz) [reply src/main/resources/application.yaml]: you can make it a flat property.
maybe `ocr-valiration-max-tries`

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -218,3 +218,6 @@ process-payments:
   enabled: ${PROCESS_PAYMENTS_ENABLED:true}
 
 use-wrapping-zip: true
+
+ocr-validation:
+  retry-count: 2
```
Comment (rkondratowicz) [inline src/main/resources/application.yaml:223]: do you plan to add any other properties under `ocr-validation`?

Comment (rkondratowicz) [reply src/main/resources/application.yaml:223]: how many times will it try to do the validation 2 or 3?

Comment (apereverzin) [reply src/main/resources/application.yaml:223]: 2 total tries

Comment (apereverzin) [reply src/main/resources/application.yaml:223]: and there will be no other props in ocr-validation - as of now, so yes, will make it flat

**Review by mdayican — APPROVED** [review]

**Review by doncem — APPROVED** [review]

**Pull request merged**